hoomd/operations.py

```diff
@@ -24,21 +59,36 @@ def __init__(self, simulation=None):
 
         self._tuners.append(ParticleSorter())
```

Comment:
They can, and it isn't dangerous, but it would significantly slow down their simulation. We expose the object because there are properties that users can usefully modify.

Reply:
I would expect that instantiating a new `Operations` instance is the same as calling `clear` on all the lists of an existing one. That is, I expect both objects to be empty. However, clearing the list of tuners will also delete the particle sorter. 😕 I recommend one or more of the following:
1. Document clearly that newly-instantiated `Operations` objects have a `ParticleSorter` in the tuners by default.
2. Ensure that a `ParticleSorter` is automatically created/added at runtime if one does not exist (or at least warn that a `ParticleSorter` is not present). This assumes that users always want one, which I think is true?

Reply:
We should document it explicitly.

Users always want to use the sorter, it always improves performance, and they should be able to change the sorter parameters before scheduling. The unit tests use this functionality to reduce the grid size and improve startup time. Users may want to tweak grid parameters or the period to potentially impact performance.

Reply:
I am not sure we want to do 2 except to warn users. If they explicitly remove the particle sorter, we should respect that.